addons/binding/org.openhab.binding.hs110/src/main/java/org/openhab/binding/hs110/internal/HS110.java

```diff
@@ -0,0 +1,134 @@
+package org.openhab.binding.hs110.internal;
+
+import static org.eclipse.smarthome.core.library.types.OnOffType.ON;
+import static org.openhab.binding.hs110.internal.Util.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.math.BigDecimal;
+import java.net.ConnectException;
+import java.net.InetAddress;
+import java.net.Socket;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+/**
+ * TP-Link HS100
+ *
+ * @author Insxnity
+ * @copyright Copyright (c) 2016, Insxnity Development
```

Comment:
> Thus the Class is a complete rewrite.

If this is the case, then the header should indeed be "our" header - but please make sure that it is a re-write and you did not do any copy&paste from the original class, because that would mean a copyright infringement.